docs/circuits.ipynb

```diff
@@ -579,6 +579,137 @@
     "Especially useful is dropping the last moment (which are often just measurements): `circuit[:-1]`, or reversing a circuit: `circuit[::-1]`.\n"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from cirq.ops import CZ, H
</pre><p>It's out of scope of this PR, but it'd be nice if we can remove these explicit imports and instead use <code>cirq.CZ</code> and <code>cirq.H</code> to be consistent with all the other tutorials (and the first part of this tutorial itself).   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5399/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
+1

Reply:
Done.